waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/client/TunnelingMetaStoreClientFactory.java

```diff
@@ -15,100 +15,45 @@
  */
 package com.hotels.bdp.waggledance.client;
 
```

Comment:
Small PR turned into a larger one when Daniel asked for test, this class was doing a lot of things, and the change which introduced the bug couldn't be tested for without decoupling this class